### PR TITLE
roundedtb: Add version R2.2

### PR DESCRIPTION
**Pull request by UnicorNora**

### CODE REVIEW

File: bucket/roundedtb.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        "script": "taskkill /F /IM RoundedTB.exe"
+    },
+    "shortcuts": [
+        [
+            "RoundedTB.exe",
+            "RoundedTB"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/torchgm/RoundedTB"
```
Comment (Ash258) [inline bucket/roundedtb.json:25]: Cannot work...

File: bucket/roundedtb.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+    "homepage": "https://github.com/torchgm/RoundedTB/",
+    "version": "R2.2",
+    "license": "GPL-3.0-only",
```
Comment (Ash258) [inline bucket/roundedtb.json:4]: No description

File: bucket/roundedtb.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+    "homepage": "https://github.com/torchgm/RoundedTB/",
+    "version": "R2.2",
```
Comment (Ash258) [inline bucket/roundedtb.json:3]: Why there is `R`??

File: bucket/roundedtb.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    "license": "GPL-3.0-only",
+    "notes": [
+        "RoundedTB will NOT automatically start when user logs in.",
+        "Open RoundedTB, right-click on the tray icon, and select 'Run on Startup' to add it to startup programs."
+    ],
```
Comment (Ash258) [inline bucket/roundedtb.json:8]: Irrelevant. User should be prompted by application.

```suggestion
```

File: bucket/roundedtb.json [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/torchgm/RoundedTB/releases/download/R2.2/RoundedTB.zip",
+            "hash": "6efa70442dc21f281e68595fd94de8652f736a3db612ef1b690eea58e3b6c1a3"
+        }
+    },
+    "uninstaller": {
+        "script": "taskkill /F /IM RoundedTB.exe"
+    },
```
Comment (Ash258) [inline bucket/roundedtb.json:17]: Does not make sense. Uninstallation will be stopped if the executable is running

```suggestion
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/roundedtb.json (hunk 3 above)
Comment (UnicorNora) [reply bucket/roundedtb.json:3]: Thats the tag on github

File: bucket/roundedtb.json (hunk 1 above)
Comment (UnicorNora) [reply bucket/roundedtb.json:22]: Why not

**Pull request merged**